packages/dm-core-plugins/src/form/fields/ObjectField.tsx

```diff
@@ -308,6 +319,7 @@ export const ObjectField = (props: TObjectFieldProps): JSX.Element => {
       label={displayLabel}
       type={type === 'object' && values ? values.type : type}
       defaultValue={defaultValue}
+      readOnly={readOnly}
```

Comment:
readOnly is already passed on L317, so there is no need for this nor the unwrapping of readonly from the props on L303